packages/orchestrator/internal/server/sandboxes.go

```diff
@@ -45,7 +46,21 @@ func (s *server) Create(ctxConn context.Context, req *orchestrator.SandboxCreate
 		attribute.String("envd.version", req.Sandbox.EnvdVersion),
 	)
 
-	metricsWriteFlag, flagErr := s.featureFlags.BoolFlag(featureflags.MetricsWriteFlagName, req.Sandbox.SandboxId)
+	// setup launch darkly
+	ctx = featureflags.CreateContext(
+		ctx,
+		ldcontext.NewBuilder(req.Sandbox.SandboxId).
+			Kind(featureflags.SandboxKind).
+			SetString(featureflags.SandboxTemplateAttribute, req.Sandbox.TemplateId).
+			SetString(featureflags.SandboxKernelVersionAttribute, req.Sandbox.KernelVersion).
+			SetString(featureflags.SandboxFirecrackerVersionAttribute, req.Sandbox.FirecrackerVersion).
+			Build(),
+		ldcontext.NewBuilder(req.Sandbox.TeamId).
+			Kind(featureflags.TeamKind).
+			Build(),
+	)
```

Comment:
This creates a launchdarkly context based on the data in the request and stores it in the context so we don't have to manually pass it around everywhere. The launchdarkly client gets it implicitly by pulling it out of the context.